src/custom/categories.js

```diff
@@ -0,0 +1,500 @@
+export const data = {
```

Comment:
have you considered if the other format was simpler to work with ? and simpler to maintain ?

Reply:
It was already 90% "done" that's why I stacked in this format.
I believe that the other format you proposed is simpler and I am very happy to change it.